package.json

```diff
@@ -186,7 +186,7 @@
         "qrcode": "~1.5.0",
         "rollup-plugin-visualizer": "^5.6.0",
         "sass": "~1.42.1",
-        "stylelint": "^15.10.1",
+        "stylelint": "^16.1.0",
```

Comment:
This is a Breaking change!

See https://stylelint.io/changelog/#1600
> Removed: Node.js less than 18.12.0 support

There are also some deprecation warnings in the current stle configuration => these need to be migrated first, as they are removed in v16

Reply:
Yea and check the check-linter run it going crazy